src/core/orderbook.js

```diff
@@ -1142,20 +1142,20 @@ OrderBook.prototype.getOffersSync = function() {
 OrderBook.prototype.toJSON =
 OrderBook.prototype.to_json = function() {
   const json = {
-    taker_gets: {
+    gets: {
```

Comment:
While the ambiguity is unlikely to be a problem the potential is there, and it's also true the RPC arguments as recognized by rippled follow a taker_* pattern, so when you add it all up it seems like we should go with taker_* ?